utils/build_swift/tests/expected_options.py

```diff
@@ -252,16 +260,25 @@ class UnsupportedOption(_BaseOption):
     """Option that is not supported."""
 
     def __init__(self, *args, **kwargs):
-        kwargs['dest'] = kwargs.get('dest')
+        kwargs['dest'] = kwargs.pop('dest', None)
```

Comment:
This is still really weird to me. What are you trying to do here?

Reply:
I'm injecting my own definition for `dest` such that the `UnsupportedOption` class does not need to be instantiated with it manually passed. Here I'm passing through any manually set `dest` or setting it by default to None. In all cases it's defined in the `kwargs` and satisfies the `_BaseOption` constructor.